explorer-front/src/routes/home.vue

```diff
@@ -196,7 +196,7 @@
                         <div class=msg>
                             <div class=msg_change>
                                 <span class="msg_change_left">24h Change : </span>
-                                <span class="msg_change_right" v-bind:class="[ market.trends == 1 ? 'red' : 'green']" >{{ market.trends == 1 ? '+' : '-' }} {{ market.change24h }}%</span>
+                                <span class="msg_change_right" v-bind:class="[ market.trends == 1 ? 'green' : 'red']" >{{ market.trends == 1 ? '+' : '-' }} {{ market.change24h }}%</span>
```

Comment:
It's an interesting change here, most stock markets use green for up and red for down, but Chinese stock market uses red for up and green for down, I think it makes sense to make this change to be consistent with most of the markets and using red for up and green for down could be a localization feature in future.

Reply:
Interesting, I had no idea that it was switched like that. Cheers